src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
I'm not sure it's a good idea to have Picard release with potentially a different version of htsjdk than is pulled down during a typical build and tested with in travis, etc. If Picard needs to release with a newer version of htsjdk than is in master, why not just do a PR to update `ivy.xml` before doing the release?